frontend/src/dialogs/SecretDialog.vue

```diff
@@ -69,9 +69,10 @@ limitations under the License.
 
           <slot name="data-slot"></slot>
           <alert color="error" :message.sync="errorMessage" :detailedMessage.sync="detailedErrorMessage"></alert>
+          <p class="subheading mt-3">Do not use personalized login credentials as they should not be used as secret.</p>
```

Comment:
Shouldn't we rather point out here to use a technical user? So something like "It is not recommended to use personalized login credentials here. Please use the credentials of a technical user."

Reply:
Yes what do the others think? @petersutter @holgerkoser

Reply:
I would phrase it as follows:
"Do not use personalized login credentials. Instead, use the credentials of a technical user.